### PR TITLE
Jorgen/laravel siri emulator

### PR DESCRIPTION
**Pull request by jbtfk**
Minor change to make emulator buildable from laravel siri repository (npx mix)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,20 +119,12 @@ The following artisan commands manages SIRI subscriptions:
...
-        'axios',
-        'vue',
-    ]);
-```
+**In laravel-siri**
+Run `npx mix`
+copy public/siri/js folder to destination project public/siri/js
+
+**In destination project**
+make test subscription with artisan
```
Comment (tfk-kaare) [inline README.md:127]: This isn't rendered as intended in markdown:

bilde

Use bullet points or forced line break (double space at EOL).

**Review by tfk-kaare — COMMENTED** [review]
Bug in markdown, otherwise it's good :-)

File: README.md (hunk 1 above)
Comment (jbtfk) [reply README.md]: Implemented both double space and bullet points now

**Review by tfk-kaare — APPROVED** [review]
Done

**Pull request merged**